### PR TITLE
RE2022-203: Add beginnings of samples API

### PR DESCRIPTION
**Pull request by MrCreosote**
just counts rows in the DB for now.

### CODE REVIEW

File: src/service/data_products/samples.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    """
+
+
+# At some point we're going to want to filter/sort on fields. We may want a list of fields
+# somewhere to check input fields are ok... but really we could just fetch the first document
+# in the collection and check the fields 
+@_ROUTER.get(
+    "/",
+    response_model=SamplesTable,
+    description="Get the corresponding sample attribues for each data unit in the collection, "
```
Comment (Tianhao-Gu) [inline src/service/data_products/samples.py:137]: *attributes

**Review by Tianhao-Gu — DISMISSED** [review]
other than the typo, LGTM. There are a few unused imports, those are there for future usage?

File: src/service/data_products/samples.py (hunk 1 above)
Comment (MrCreosote) [reply src/service/data_products/samples.py]: fixed

**Comment by MrCreosote** [comment]
> There are a few unused imports, those are there for future usage?

Maybe, but probably also because I've tried to get linting working several times in VSC (which is something that should Just Work, the amount of JSON based config you have to do is ridiculous) and can never get it to recognize imports in the same project (and some other quirks). On the last PR for samples I'll clean up any extraneous imports.

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**